components/header/web3Connect/Account.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { Button, createStyles, Modal, Tooltip } from '@mantine/core';
+import { shortenAddress, useConfig, useEtherBalance, useEthers, useLookupAddress } from '@usedapp/core';
+import { formatEther } from '@ethersproject/units'
+
+const useStyles = createStyles(theme => ({
+  accountInfo: {
+    display: "block",
+    padding: 16,
+    borderRadius: theme.radius.md,
+    boxShadow: `2px 2px 2px 2px ${theme.colors.gray[4]}`
+  },
+  modalTitle: {
+    fontSize: theme.fontSizes.xl,
+    lineHeight: 0,
+  }
+}))
+
+const Account = () => {
+  const { classes } = useStyles()
+  const { account, chainId } = useEthers()
+  const config = useConfig()
+  const ens = useLookupAddress()
+  const [showModal, setShowModal] = useState(false)
+  const etherBalance = useEtherBalance(account)
+
+
+  if (chainId && config.networks && config.networks.every((chain) => chain.chainId !== chainId)) {
```

Comment:
```suggestion
  if (chainId && !config?.networks.includes(chainId)) {
```

Reply:
`chainId` is number, while `config.networks` contains array of `Chain` objets

Reply:
Ah sorry didn't notice, but still this is more readable IMHO.
```suggestion
!config?.networks.map(({chainId}) => chainId).includes(chainId)
```